examples/support/mock_data.py

```diff
@@ -163,39 +225,60 @@ def _generate_galaxy_catalog(cluster_m, cluster_z, cluster_c, cosmo, ngals, Delt
     return galaxy_catalog['ra', 'dec', 'e1', 'e2', 'z', 'ztrue']
 
 
-def _draw_source_redshifts(zsrc, cluster_z, zsrc_min, zsrc_max, ngals):
+def _draw_source_redshifts(zsrc, zsrc_min, zsrc_max, ngals):
     """Set source galaxy redshifts either set to a fixed value or draw from a predefined
     distribution. Return a table (GCData) of the source galaxies
 
     Uses a sampling technique found in Numerical Recipes in C, Chap 7.2: Transformation Method.
     Pulling out random values from a given probability distribution.
+    
+    Parameters
+    ----------
+    ngals : float
+        Number of galaxies to generate
+    zsrc : float or str
+        Choose the source galaxy distribution to be fixed or drawn from a predefined distribution.
+        float : All sources galaxies at this fixed redshift
+        str : Draws individual source gal redshifts from predefined distribution. Options
+              are: chang13
+    zsrc_min : float
+        The minimum source redshift allowed.
+    zsrc_max : float, optional
+        If source redshifts are drawn, the maximum source redshift
+
+    Returns
+    -------
+    galaxy_catalog : clmm.GCData
+        Table of true and 'measured' redshifts, which here the same. Photometric errors
```

Comment:
It would be better to add "redshift" between "photometric" and "errors".

Reply:
Will correct this in #306